keel-sql/src/main/resources/db/changelog/20200106-tie-artifacts-to-delivery-configs.yml

```diff
@@ -0,0 +1,91 @@
+databaseChangeLog:
+  - changeSet:
+      id: tie-artifacts-to-delivery-configs
+      author: emjburns
+      changes:
+        - addColumn:
+            tableName: delivery_artifact
+            columns:
+              - column:
+                  name: delivery_config_name
+                  type: varchar(255)
+                  value: "keel"
```

Comment:
Default values here which are def garbage at first, but it should be easy enough to go back and correct them by hand.

Reply:
Not sure if this makes sense over a nullable column without a default value.

Reply:
I toyed with that too. I'm not sure what is best - but, we really don't want to allow this to be null. An artifact should be associated with a delivery config. So I felt like the initial weirdness was an ok thing for where we are in adoption.

Reply:
Should have a not null constraint then

Reply:
Is keel a valid delivery config name? I feel like it should be a reserved value that's a little more explicit (i.e. "_pending" if there's validation that delivery config names have to start with a-zA-Z0-9)

Reply:
Hmm. This will only apply for existing values, are you worried that it's not explicit enough and easy to miss that it's garbage data?